internal/cmd/skupper/connector/nonkube/connector_delete_test.go

```diff
@@ -147,17 +145,40 @@ func TestCmdConnectorDelete_Run(t *testing.T) {
 
 		t.Run(test.name, func(t *testing.T) {
 
+			createConnectorResource(path, t)
 			cmd.connectorName = test.deleteName
 			cmd.namespace = test.namespace
 			cmd.connectorHandler = fs.NewConnectorHandler(cmd.namespace)
 			cmd.InputToOptions()
 
 			err := cmd.Run()
 			if err != nil {
```

Comment:
Instead of looking at the `err` condition, better if the tests checks a `test.errorMessage` is expected.